src/frontend/js/types/Joanie.ts

```diff
@@ -122,6 +122,10 @@ export enum OrderState {
 export interface Order {
   id: string;
   course: string;
+  // I wanted to use the `course` attribute, but it is already in use for the course's id.
+  // Maybe would it makes more sens to rename `course` to `courseId` in order to use `course`
```

Comment:
If we need to have more information than id for the course, the api could return these information instead of a simple id.

`course: Course`

Reply:
I introduced a new attribute `courseEntity?: Course;` because I was wondering that maybe changing the type of `course` from `string` to `Course` would break something somewhere. 

But indeed it seems that the `course` attribute is not used anywhere, could you confirm my thought ?

Reply:
Yes I confirm ! `order.course` is currently not used so you can change the type of this property.